project_national/national_upgrades_HPs_TMY3.yml

```diff
@@ -0,0 +1,156 @@
+      include_timeseries_component_loads: true
+      include_timeseries_unmet_hours: true
+      include_timeseries_zone_temperatures: true
+      include_timeseries_airflows: true
+      include_timeseries_weather: true
+      include_timeseries_resilience: true
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: true
+      retain_schedules_csv: false
+
+kestrel:
+  n_jobs: 10
+  minutes_per_sim: 5
+  account: heatpump
+  postprocessing:
+    time: 80
+    n_workers: 1
+  sampling:
+    time: 20
+
+postprocessing:
+  keep_individual_timeseries: true
+  # partition_columns:
+  #   - State
+  #   - County
+  aws:
+    region_name: us-west-2
+    s3:
+      bucket: heat-pump-deployment
+      prefix: test_runs
+    athena:
+      glue_service_role: service-role/AWSGlueServiceRole-default
+      database_name: heatpumpdeployment
+      max_crawling_time: 1200
+
+
+baseline:
+  n_buildings_represented: 139647020 # American Community Survey 2021 5-year, B25001, does not include territories ( 138765649 without AK and HI )
+
+upgrades:
+  # Case 1: Baseline Stock - AMY2018, Furnace over-sizing factor = 1.7 for homes with a ducted fossil fuel furnace, default cooling autosizing factor (1.0)
+  #         Updated defrost approach, tweak sampled heat pumps
+  - upgrade_name: Case 3 - Baseline Upgrade - EnergyStar Heat Pump, Electric Backup
+
+    package_apply_logic: &remove_non_fossil_non_elec_fuels
+      - not: Heating Fuel|Wood
+      - not: Heating Fuel|Other Fuel
+      - not: Heating Fuel|None
+
+    options:
+      - &hvac_heating_efficiency_ashp_case_3
+        option: HVAC Heating Efficiency|ASHP, SEER 16.0, 9.17 HSPF, Case 3
+        apply_logic: &ducted_ASHP_apply_logic
+          - HVAC Has Ducts|Yes
+        costs: &dummy_cost_values
+          - value: 999 # dummy value
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &dummy_lifetime_value 30
+      - &hvac_cooling_efficiency_ducted_heat_pump
+        option: HVAC Cooling Efficiency|Ducted Heat Pump
+        apply_logic: *ducted_ASHP_apply_logic
+      - &hvac_distribution_ductwork_restriction
+        option: HVAC Distribution|Ductwork Restriction
+        apply_logic: *ducted_ASHP_apply_logic
+
+      - &hvac_heating_efficiency_mshp_case_3
+        option: HVAC Heating Efficiency|MSHP, SEER 16.84, 9.64 HSPF, Case 3
+        apply_logic: &nonducted_MSHP_apply_logic
+          - HVAC Has Ducts|No
+          - not: HVAC Heating Efficiency|MSHP, SEER 29.3, 14 HSPF
+        costs: *dummy_cost_values
+        lifetime: *dummy_lifetime_value
+      - &hvac_cooling_efficiency_non_ducted_heat_pump
+        option: HVAC Cooling Efficiency|Non-Ducted Heat Pump
+        apply_logic: *nonducted_MSHP_apply_logic
+
+  - upgrade_name: Case 2 - Baseline Upgrade - EnergyStar Heat Pump, Existing System Backup
+
+    package_apply_logic: *remove_non_fossil_non_elec_fuels
```

Comment:
@jonwinkler If the existing heating system is electric, should this upgrade apply?

Reply:
@joseph-robertson I think we discussed this case with @whiphi92 last week and he mentioned that if the existing system is electric, we tend not to apply this upgrade. I think it would yield the same result as an all-electric heat pump upgrade.

Reply:
Right, it would. But just checking since "dual-fuel" is in the option name; just wanted to make sure you weren't expecting a "skip" for these datapoints...